app/src/components/inputs/inputRadio/inputRadio.jsx

```diff
@@ -33,6 +33,7 @@ export const InputRadio = ({
   mobileDisabled,
   title,
   mode,
+  small,
```

Comment:
Maybe better using `size` as props to be more flexible between different input sizes like small-medium-large?